designer/client/src/i18n/translations/en.translation.json

```diff
@@ -88,11 +88,13 @@
     "contentfields_info": "Content components allow you to add information to your page",
     "create": "Add component",
     "create_info": "Select a component to add to your page. If your page has only one component, the component title does not show on the page. To name or describe the component, use the page title instead.",
-    "edit": "Edit {{name}} component",
+    "edit": "Edit {{name, lowerFirst}} component",
     "inputfields_info": "Input fields allow you to ask users to enter information",
     "selectfields_info": "Selection fields allow you to provide lists of answers for users to choose from",
-    "move_up": "Move component up",
-    "move_down": "Move component down"
+    "moveUp": "Move component up",
+    "moveUp_label": "Move {{name, lowerFirst}} component up",
+    "moveDown": "Move component down",
+    "moveDown_label": "Move {{name, lowerFirst}} component down"
```

Comment:
@davidjamesstone I've followed the same screen reader approach as the component icons

But the mouseover `title` attribute "Move component up" or "Move component down" is unchanged

That look alright?